tests/test_AllocateSizeMoreThanClusterSize.py

```diff
@@ -0,0 +1,240 @@
+import json
+from ocs import ocp
+from openshift.dynamic import DynamicClient
+from utility import utils, templating
+from kubernetes import config
+from ocsci.enums import StatusOfTest
+import time
+
+k8s_client = config.new_client_from_config()
+dyn_client = DynamicClient(k8s_client)
+log = logging.getLogger(__name__)
+
+PVC_YAML = os.path.join("templates/ocs-deployment",
+                        "PersistentVolumeClaim.yaml")
+POD_YAML = os.path.join("templates/ocs-deployment",
+                        "Pod.yaml")
+SC_YAML = os.path.join("templates/ocs-deployment",
+                       "StorageClass.yaml")
+TEMP_PVC_YAML_FILE = '/tmp/pvc_test.yaml'
+TEMP_POD_YAML_FILE = '/tmp/pod_test.yaml'
+TEMP_SC_YAML_FILE = '/tmp/sc_test.yaml'
+
+POD = ocp.OCP(kind='Pod')
+PVC = ocp.OCP(kind='PersistentVolumeClaim')
+PV = ocp.OCP(kind='PersistentVolume')
+SC = ocp.OCP(kind='StorageClass')
+NAMESPACE = ocp.OCP(kind='Project')
+
+
+def ceph_storage_capacity(ceph_tool):
+    '''
+    Returns the total capacity of the ceph cluster
+    in openshift-storage project
+    '''
+
+    # TODO: use an exec function to run ceph status
+    pods = run_cmd(f'oc rsh -n openshift-storage {ceph_tool} \
+    ceph status --format json')
+    pods_json = json.loads(pods)
+    avail_cap = pods_json['pgmap']['bytes_total']
+    avail_cap_gb = avail_cap / (1024 * 1024 * 1024)
+    return avail_cap_gb
+
+
+def find_cephtool():
+    '''
+    Returns rook-ceph-tools pod from the list of pods in
+    openshift-storage project
+    '''
+
+    v1_pods = dyn_client.resources.get(api_version='v1', kind='Pod')
+    out = v1_pods.get(
+        namespace='openshift-storage', label_selector='app=rook-ceph-tools'
+    )
+    return out['items'][0]['metadata']['name']
+
+
+def create_namespace(**kwargs):
+    '''
+    Creates a project if it is not already available
+    '''
+    project_name = (kwargs['project_name'])
+    project_get = NAMESPACE.get()
+    namespaces = []
+    for i in range(len(project_get['items'])):
+        namespaces.append(project_get['items'][i]['metadata']['name'])
+    log.info(f'checking id project {project_name} already exists')
+    if project_name in namespaces:
+        log.info(
+            f'project {project_name} exists, using the existing namespace'
+        )
+        return True
+    else:
+        log.info(f'creating a new project {project_name}')
+        return run_cmd(f'oc new-project {project_name}')
+
+
+def delete_namespace(**kwargs):
+    '''
+    Deletes a project
+    '''
+
+    project_name = kwargs['project_name']
+    return run_cmd(f'oc delete project {project_name}')
+
+
+def create_storageclass(**kwargs):
+    '''
+    Create a storage class
+    '''
+
+    sc_name = (kwargs['sc_name'])
+    sc_get = SC.get()
+    storage_classes = []
+    for i in range(len(sc_get['items'])):
+        storage_classes.append(sc_get['items'][i]['metadata']['name'])
+    log.info(f'checking if {sc_name} exists already')
```

Comment:
ok, so as a general note:
You don't need to check every time before creating a resource if it is already there or not.
In case it is there, we need to fix some other test that caused the leftover